webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -320,17 +384,29 @@ const RepositorySetsTab = () => {
                           {
                             title: __('Override to disabled'),
                             isDisabled: isOverridden && !isEnabled,
-                            onClick: () => overrideToDisabled({ labels: [label] }),
+                            onClick: () => updateOverrides({
+                                              enabled: false,
+                                              search: repoSetSearchQuery(id),
+                                              singular: true,
```

Comment:
These function calls feel too closely coupled to the UI. I think I'd prefer extracting them to functions, something like
```js
// earlier in this file
const disableRepoSet = (search) => updateOverrides({
                                              enabled: false,
                                              search,
                                              singular: true,
                                          });
const disableRepoSets = (search) => updateOverrides({
                                              enabled: false,
                                              search,
                                              singular: false,
                                          });
// ... and so on
```
and then here it would be 
```js
onClick: disableRepoSet(repoSetSearchQuery(id)),
```